pkg/driver/node.go

```diff
@@ -69,15 +71,21 @@ func (d *driver) NodeStageVolume(_ context.Context, req *csi.NodeStageVolumeRequ
 	}
 
 	var options []string
-
 	if readOnly {
 		options = append(options, "ro")
 	} else {
 		options = append(options, "rw")
 	}
 	options = append(options, mountOptions...)
+
+	var formatOptions []string
+	mkfsOptions, exists := volumeContext[ParameterMkfsOptions]
```

Comment:
Can we use `ok` instead of `exists` as this is typically how you call the variable in go.

Reply:
Done